Simplenote.xcodeproj/xcshareddata/xcschemes/SimplenoteScreenshots.xcscheme

```diff
@@ -11,6 +11,24 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
+      <PreActions>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "Configure simulator status bar"
+               scriptText = "# I tried to extract this in a dedicate script and to execute it from here, but it doesn&apos;t run...&#10;IPHONE_TIME=$(ruby -e &quot;require &apos;time&apos;;puts Time.new(2007, 1, 9, 9, 42, 0).iso8601&quot;)&#10;&#10;xcrun simctl boot &quot;${TARGET_DEVICE_IDENTIFIER}&quot;&#10;&#10;xcrun simctl status_bar &quot;${TARGET_DEVICE_IDENTIFIER}&quot; override \&#10;    --time $IPHONE_TIME \&#10;    --dataNetwork wifi \&#10;    --wifiMode active \&#10;    --wifiBars 3 \&#10;    --cellularMode active \&#10;    --batteryState charged \&#10;    --batteryLevel 100&#10;">
```

Comment:
> I tried to extract this in a dedicate script and to execute it from here, but it doesn't run...

😞 

I tried `source`ing it, `sh "${SRC_ROOT}/path/to/script"` and other combos. No luck.

Any ideas? I'd *really* like to have this out in its own script.

Reply:
It worked for me locally with `sh ${PROJECT_DIR}/path/to/script` (instead of `${SRC_ROOT}`). Maybe give that a try?

Reply:
YES! Thanks!

![home alone 2 kevin saying yes in the street of NYC](https://media.giphy.com/media/63IzeZiFZOJLXCfFWR/giphy.gif)